### ISSUE TITLE
Incorporate drainage area into rain garden / bioswale eco benefit calculation

### ISSUE THREAD

**Issue by jwalgran** [OpenTreeMap/otm-core#2689]
The drainage area is a new field added in https://github.com/OpenTreeMap/otm-core/issues/2688

The new proposed equation is:

Annual rainfall x area of polygon x square footage of area that drains to the rain garden x constant number = Annual stormwater diverted

Drainage area is not a required field so when calculating aggregate benefits we need to skip over features that do not have a value, and reflect that in the "basis."

**Comment by dboyer** [comment]
Perhaps instead:

Annual rainfall x (area of polygon + (square footage of area that drains to the rain garden x constant number ))

**Comment by dboyer** [comment]
The client confirmed that is the correct equation.

**Comment by dboyer** [comment]
The constnat number should be 0.85

**Comment by RickMohr** [comment]
Just to clarify, you mean the default value for the "Percent Stormwater Diverted" constant should be 0.85, but users should be able to change it, right?

**Comment by dboyer** [comment]
Correct. I'm guessing most people will not change the default, but editing it should be an option for map owners. 

I'm working on confirmation of what we should call that field. Civil engineers sometimes refer to it as the "runoff coefficient."

**Comment by RobinIsTheBird** [comment]
Request for clarification:

@dboyer I presume that the client confirmed mmaurizi's equation, right?

On the Green Infrastructure management page, we are presenting the number as 0..100, although internally it is converted to a fraction. Is this the desired display representation?

Last but not least, are we ready to commit to calling it "runoff coefficient" yet?

**Comment by dboyer** [comment]
Just received confirmation that we can call it "runoff coefficient." The equation we proposed (below) is correct.

Annual rainfall x (area of polygon + (square footage of area that drains to the rain garden x constant number ))

Displaying it as 0.85 on the management page would be preferable as that's how it is displayed as part of something called the "Rational Equation Method" - http://www.lmnoeng.com/Hydrology/rational.php - which is often used for this type of GSI calculation.
